FirelandsProject/cataclysm
# Trinity string entry -106 not found in DB.

### Changes proposed

- Remove the emote -106 in the npc script, because this number does not exist in trinity_string and generates an error message in the game client.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10
- In-game

### How to Test the Changes:


1. Create a worgen, and in the second mission, when you have to eliminate 6 of the npc in question, when you enter enrage, you will see an error message in the game client chat.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
